### PR TITLE
[AMO-128510] - Define Attributed Conversions Schema

### PR DESCRIPTION
**Pull request by jishah-adobe**
Please link to the issue #…

### CODE REVIEW

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+        },
+        "xdm:viewthroughValue": {
+          "title": "View Through Value",
+          "type": "number",
+          "description": "Raw Conversion Value if Triggering Event Type is Impression"
+        },
+        "xdm:viewthroughCount": {
+          "title": "View Through Count",
+          "type": "number",
+          "description": "Raw Conversion Count if Triggering Event Type is Click"
```
Comment (vvaidyan-adobe) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: event type is impression I believe

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      "properties": {
+        "xdm:triggerType": {
+          "title": "Trigger Event Type",
+          "type": "string",
+          "description:": "Trigger Type for an Event - examples include click, impression"
+        },
+        "xdm:pathPosition": {
+          "title": "Path Position",
+          "type": "string",
+          "description": "Path Position to conversion"
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: Description is not intuitive. Can you please elaborate?

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Trigger Time of the Event"
+        },
+        "xdm:originaltriggerTime": {
+          "title": "Original Trigger Time",
+          "type": "string",
+          "format": "date-time",
+          "description": "Original Trigger Time"
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: What's the difference between TriggerTime and OriginalTriggerTime?

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name."
+        },
+        "xdm:displaycampaignId": {
+          "title": "Display Campaign Identifier",
+          "type": "string",
+          "description": "Campaign ID on the display advertising platform."
+        },
+        "xdm:campaignId": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: Should we rename it to searchCampaignId?

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Advertiser Identifier",
+          "type": "string",
+          "description": "Advertiser Identifier"
+        },
+        "xdm:searchEngineId": {
+          "title": "Search Engine Identifier",
+          "type": "integer",
+          "description": "The application-specified identifier used to identify the Search Advertising Platform Name."
+        },
+        "xdm:displaycampaignId": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: If you are camelCase, change it to "displayCampaignId"

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Ad Group Identifier",
+          "type": "string",
+          "description": "Ad Group ID on the search advertising platform."
+        },
+        "xdm:displayadId": {
+          "title": "Display Ad Identifier",
+          "type": "string",
+          "description": "Ad ID on the display advertising platform."
+        },
+        "xdm:adId": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: change field id to searchAdId? would there be a situation where both searchAdId and displayAdId present with a conversion? If not, can we merge them to adId?

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Domain Placement",
+          "type": "string",
+          "description": "Placement Domain"
+        },
+        "xdm:referrerUrl": {
+          "title": "Referrer URL",
+          "type": "string",
+          "description": "The referrer is the webpage that sends visitors to your site using a link."
+        },
+        "xdm:searchTerm": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: What is the different between searchTerm and term? Is this user entered search term where as the term is the what network used to match to an ad?

File: extensions/adobe/experience/adcloud/attributedconversions.schema.json [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+          "title": "Search Term",
+          "type": "string",
+          "description": "Original Term used for the Search"
+        },
+        "xdm:geoiplocationId": {
```
Comment (udayeyunni) [inline extensions/adobe/experience/adcloud/attributedconversions.schema.json]: Do you want to camelCase here?

**Comment by prabhum2** [comment]
@jishah-adobe  - is this PR still required ?

**Comment by jishah-adobe** [comment]
This may not be required now but at a later time. Closing this PR

**Pull request closed without merging**